### PR TITLE
gsoc: Add docs for apply single action.

### PR DESCRIPTION
**Pull request by Nosferatul**

### CODE REVIEW

**Comment by jayvdb** [comment]
Commit message isnt descriptive of the changes

**Comment by jayvdb** [comment]
This PR is not "Add docs", as if it is some optional improvement.  This PR is fixing _bugs_.  It was a bug to not include a param description when adding a parameter.

**Comment by Nosferatul** [comment]
ok, got it, thx

**Comment by jayvdb** [comment]
Create an issue describing the bug, and add "Fixes ....." to your commit message.

**Comment by Nosferatul** [comment]
it's ok now @jayvdb ?

**Comment by yukiisbored** [comment]
@Nosferatul Please fix the commit message, it doesn't follow the commit message guidelines. coala.io/commit

File: coalib/output/ConsoleInteraction.py [hunk 1]
```diff
@@ -154,6 +154,8 @@ def acquire_actions_and_apply(console_printer,
     :param result:          A derivative of Result.
     :param file_dict:       A dictionary containing all files with filename as
                             key.
+    :param apply_single:    The action that should be applied for all results.
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Can you please have a better commit message?

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -154,6 +154,8 @@ def acquire_actions_and_apply(console_printer,
...
+                            False if it's not use.
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: In use/used

File: coalib/output/ConsoleInteraction.py [hunk 3]
```diff
@@ -266,6 +268,8 @@ def print_result(console_printer,
...
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Same as earlier

File: coalib/output/ConsoleInteraction.py [hunk 4]
```diff
@@ -154,6 +154,8 @@ def acquire_actions_and_apply(console_printer,
     :param result:          A derivative of Result.
     :param file_dict:       A dictionary containing all files with filename as
                             key.
+    :param apply_single:    The action that should be applied for all results.
+                            False if it's not used.
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: The commit shortlog still stinks. Try:
Apply_single: Add documentation for param

File: coalib/output/ConsoleInteraction.py [hunk 5]
```diff
@@ -266,6 +268,8 @@ def print_result(console_printer,
...
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: Better phrasing: "If it's not selected, has a value of `False`".

**Comment by Adrianzatreanu** [comment]
Ack fbb71a1

**Comment by Nosferatul** [comment]
@rultor merge

**Pull request merged**